extras/dgoss/dgoss

```diff
@@ -25,9 +25,26 @@ run(){
     [[ -e "${GOSS_FILES_PATH}/goss.yaml" ]] && cp "${GOSS_FILES_PATH}/goss.yaml" "$tmp_dir"
     [[ -e "${GOSS_FILES_PATH}/goss_wait.yaml" ]] && cp "${GOSS_FILES_PATH}/goss_wait.yaml" "$tmp_dir"
     [[ ! -z "${GOSS_VARS}" ]] && [[ -e "${GOSS_FILES_PATH}/${GOSS_VARS}" ]] && cp "${GOSS_FILES_PATH}/${GOSS_VARS}" "$tmp_dir"
-    info "Starting docker container"
-    id=$(docker run -d -v "$tmp_dir:/goss" "${@:2}")
-    docker logs -f "$id" > "$tmp_dir/docker_output.log" 2>&1 &
+
+    # Switch between mount or cp files strategy
+    GOSS_FILES_STRATEGY=${GOSS_FILES_STRATEGY:="mount"}
+    case "$GOSS_FILES_STRATEGY" in
+      mount)
+        info "Starting docker container"
+        id=$(docker run -d -v "$tmp_dir:/goss" "${@:2}")
+        docker logs -f "$id" > "$tmp_dir/docker_output.log" 2>&1 &
+        ;;
+      cp)
+        info "Creating docker container"
+        id=$(docker create "${@:2}")
+        info "Copy goss files into container"
+        docker cp $tmp_dir $id:/goss
+        info "Starting docker container"
+        docker start $id > /dev/null
+        ;;
+      *) error "Wrong goss files strategy used! Correct options are \"mount\" of \"cp\"."
```

Comment:
Yep just a typo: I meant `of` vs `or`.

Whatever your preference is for `copy` vs `cp` ;)